### DISCUSSION TITLE
Add support for tags (currently a bug exists when importing passwords from 1password)

### DISCUSSION THREAD

**Discussion by LordVZlomka** [dani-garcia/vaultwarden#4331] [Ideas]
In addition to having the ability to create folders, it would be beneficial to assign tags. For example, I would add a tag "2FA" or "no2FA" to my accounts to quickly check which accounts are fully secured. There are a million ways this can be used to organize and fast search.

Also! If you import your passwords from 1Password, and they contain tags - VauldWarden creates a new folder to match each tag (tags are removed, and folders are created).

**Comment by BlackDex** [comment]
If I'm correct that seems to be a work in progress by Bitwarden.
This project isn't maintaining the clients. Since a feature like this needs client support, it's not something we can add.
